### DISCUSSION TITLE
Add Publication to AI2-THOR Publications!

### DISCUSSION THREAD

**Discussion by mattdeitke** [allenai/ai2thor#678] [Show and tell]
If there is a paper that uses AI2-THOR for experiments, but is not listed on [AI2-THOR Publications](https://ai2thor.allenai.org/publications/), we'd love to add it!

## Submission

Simply comment either the **Semantic Scholar link**, such as:
https://www.semanticscholar.org/paper/Target-driven-visual-navigation-in-indoor-scenes-Zhu-Mottaghi/7af7f2f539cd3479faae4c66bbef49b0f66202fa

or an **ArXiv link**, such as:
https://arxiv.org/abs/1609.05143

**Video.** _Optionally_, if there is a video, please provide the link (e.g., on YouTube/Vimeo), and we can add that as well!

**Comment by anwesanpal** [comment]
Hello,

Here are the details of our CoRL 2020 spotlight paper which has experiments conducted on AI2-THOR.

Semantic scholar - https://www.semanticscholar.org/paper/Learning-hierarchical-relationships-for-object-goal-Qiu-Pal/6d54a0000edc5b37e6314d05d502fca9f422d187

ArXiv link - https://arxiv.org/abs/2003.06749

Video - https://www.youtube.com/watch?v=eCxWwohbOd8

Website (with code) - https://sites.google.com/eng.ucsd.edu/mjolnir

**Reply by mattdeitke** [reply]
Thanks, very cool! Just added.

Weirdly the arXiv version isn't linked to the Semantic Scholar. That is why Semantic Scholar isn't show the paper's images or link to a .pdf on arXiv (which is why it was likely missed the first time). I filed an issue with them about it, so hopefully they'll link the paper to arXiv soon.

Also, code and the website links will probably be added as separate buttons soon, so I'll keep those links stored for when that's ready!

**Comment by leefree-GIT** [comment]
Hello,

Here are the details of our Neurocomputing which has experiments conducted on AI2-THOR.

https://www.sciencedirect.com/science/article/pii/S0925231221004707

**Reply by mattdeitke** [reply]
Awesome, thanks! It's been added.

It looks like Semantic Scholar wasn't showing the metadata for the paper (e.g., Figures/References), which is why it was missed on the first pass:
https://api.semanticscholar.org/CorpusID:233530501

I've pinged them about it, so hopefully they can resolve that soon :)

**Comment by sx-zhang** [comment]
Hello, Please add our work (Hierarchical Object-to-Zone Graph for Object Navigation, accepted by ICCV21) which conducts experiments on both AI2THOR and RoboTHOR. Thanks！:)
Here are some details of our work:
ArXiv link: https://arxiv.org/abs/2109.02066
Video link: https://drive.google.com/file/d/1UtTcFRhFZLkqgalKom6_9GpQmsJfXAZC/view?usp=sharing
Github link: https://github.com/sx-zhang/HOZ

**Reply by mattdeitke** [reply]
Thanks @sx-zhang,

This has been added :)

**Comment by pradipcyb** [comment]
Hi! Thanks for your great efforts. Please add our paper on object disambiguation which just got published in IEEE RA-L.
Semantic Scholar [https://www.semanticscholar.org/paper/DoRO%3A-Disambiguation-of-Referred-Object-for-Agents-Pramanick-Sarkar/8fa1efde81a80ddba4c62fbf3eac11c0d926abda](https://www.semanticscholar.org/paper/DoRO%3A-Disambiguation-of-Referred-Object-for-Agents-Pramanick-Sarkar/8fa1efde81a80ddba4c62fbf3eac11c0d926abda)
ArxiV: [https://arxiv.org/pdf/2207.14205.pdf](https://arxiv.org/pdf/2207.14205.pdf)

**Comment by guydav** [comment]
Hello, we used AI2-THOR to collect data from human participants for this work: https://www.semanticscholar.org/paper/Creativity%2C-Compositionality%2C-and-Common-Sense-in-Davidson/0dabe913442a87bb36b693de4793fa0799f55206 (and in a some forthcoming follow-ups).

**Comment by YueYANG1996** [comment]
Hi Matt,

Could you please add Holodeck to the AI2-THOR publications? 
Arxiv: https://arxiv.org/pdf/2312.09067
Semantic Scholar: https://www.semanticscholar.org/paper/Holodeck%3A-Language-Guided-Generation-of-3D-Embodied-Yang-Sun/1dbc2cdcae3e17c3d721d12a5a2d98ced727681a

Thanks!
Yue
